pkgs/development/python-modules/mitmproxy/default.nix

```
@@ -43,14 +43,14 @@
 
 buildPythonPackage rec {
   pname = "mitmproxy";
-  version = "5.3.0";
+  version = "6.0.2";
   disabled = isPy27;
```

```suggestion
  disabled = pythonAtLeast "3.8";
```
see https://github.com/mitmproxy/mitmproxy/blob/master/setup.py#L65

---

Actually,
```suggestion
  disabled = pythonOlder "3.8";
```